getting-started/main.cpp

```diff
@@ -187,7 +193,7 @@ static void encrypt_with_symmetric_ciphers(const uint8_t *key, size_t key_len)
     psa_status_t status;
     psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
     psa_algorithm_t alg = PSA_ALG_CBC_NO_PADDING;
-    uint8_t plaintext[block_size] = SOME_PLAINTEXT;
+    static const uint8_t plaintext[block_size] = SOME_PLAINTEXT;
```

Comment:
I'd recommend adding this static in a separate commit, for clarity and easier rework. It's not a change related to enabling "SHA-256 hash for PKCS1v15 message signing"

Reply:
Done